packages/api-kit/src/SafeApiKit.ts

```diff
@@ -316,9 +318,8 @@ class SafeApiKit {
     }
     const { address: delegate } = this.#getEip3770Address(delegateAddress)
     const { address: delegator } = this.#getEip3770Address(delegatorAddress)
-    const totp = Math.floor(Date.now() / 1000 / 3600)
-    const data = delegate + totp
-    const signature = await signer.signMessage(data)
+    const signature = await signDelegate(signer, delegate, this.#chainId)
+
     const body: any = {
```

Comment:
This is SafeDelegateResponse right?

Reply:
```suggestion
    const body: SafeDelegateResponse = {
```

Reply:
I think that is the request body which is marked as any on the type.